battleship.py

```diff
@@ -145,7 +164,20 @@ def drawGrid(data, canvas, grid, showShips):
 Returns: bool
 '''
 def isVertical(ship):
-    return
+    x = []
+    for row in range(len(ship)):
+        if ship[0][1] == ship[row][1]:
+            x.append(ship[row][0])
+        else :
+            return False
+    x = sorted(x)
+    for i in range(1,len(x)):
+        if (x[i] == x[i-1]+1):
+            continue
+        else :
+            return False

```

Comment:
Use not equal to avoid using continue in the if statement

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py